haiku/_src/embed_test.py

```diff
@@ -73,6 +73,39 @@ def test_inconsistent_creation_args(self):
     with self.assertRaisesRegex(ValueError, "supplied but the `embed_dim`"):
       embed.Embed(embedding_matrix=_EMBEDDING_MATRIX, embed_dim=5)
 
+  @parameterized.parameters(
+    itertools.product(["ARRAY_INDEX", "ONE_HOT"]))
+  @test_utils.transform_and_run
```

Comment:
No need for itertools.product here, or in `test_embed_invalid_lookup`.

Reply:
Ah- I agree with that. Fixed!